docs/ref/contrib/gis/forms-api.txt

```diff
@@ -153,16 +153,14 @@ Widget classes
     ``template_name`` is ``gis/openlayers.html``.
 
     ``OpenLayersWidget`` and :class:`OSMWidget` use the ``openlayers.js`` file
-    hosted on the ``openlayers.org`` website. This works for basic usage
-    during development, but isn't appropriate for a production deployment as
-    ``openlayers.org/api/`` has no guaranteed uptime and runs on a slow server.
-    You are therefore advised to subclass these widgets in order to specify
-    your own version of the ``openlayers.js`` file in the ``js`` property of
-    the inner ``Media`` class (see :ref:`assets-as-a-static-definition`). You
-    can host a copy of ``openlayers.js``
-    `tailored to your needs`_ on your own server or refer to a copy from a
-    content-delivery network like https://cdnjs.com/. This will also allow
-    you to serve the JavaScript file(s) using the ``https`` protocol if needed.
+    hosted on the ``cdnjs.cloudflare.com`` content-delivery network.
+    You can subclass these widgets in order to specify your own version of the
+    ``openlayers.js`` file `tailored to your needs`_ in the ``js`` property of
+    the inner ``Media`` class (see :ref:`assets-as-a-static-definition`).
+
+    When using :class:`OSMGeoAdmin` (derived from :class:`GeoModelAdmin`), you
```

Comment:
You need to use the full path in the `:class:` annotations to fix the broken links that are causing the docs build failure.